nova_vm/src/engine/small_integer.rs

```diff
@@ -3,7 +3,7 @@
 // file, You can obtain one at https://mozilla.org/MPL/2.0/.
 
 /// 56-bit signed integer.
-#[derive(Clone, Copy, PartialEq)]
+#[derive(Clone, Copy, PartialEq, Eq, Hash)]
```

Comment:
**nitpick:** A manual Hash impl that takes out the i64 and hashes that would probably be a good idea.

Reply:
Done.